drivers/CoreTouchSensor/include/CoreTouchSensor.h

```diff
@@ -0,0 +1,43 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "drivers/interfaces/InterfaceDigitalIn.h"
+#include "drivers/interfaces/InterfaceDigitalOut.h"
+
+#include "interface/drivers/QDAC.h"
+
+namespace leka {
+
+enum class PowerMode : uint8_t
+{
+	low,
+	normal
+};
+
+class CoreTouchSensor
+{
+  public:
+	explicit CoreTouchSensor(mbed::interface::DigitalIn &detect_pin, mbed::interface::DigitalOut &power_mode_pin,
+							 interface::QDAC &dac, Channel channel)
+		: _detect_pin(detect_pin), _power_mode_pin(power_mode_pin), _sensitivity_pin({dac, channel})
+	{
+		_detect_pin.mode(PinMode::PullUp);
+		_sensitivity_pin.dac.init();
+	}
+	auto read() -> int;
+	void setPowerMode(PowerMode power_mode);
```

Comment:
Je reviens après un long moment où je ne me suis pas documenté sur les capteurs de Toucher: je comprends le concept de lecture et d'ajuster la sensibilité, mais c'est quoi un `PowerMode` pour un capteur de toucher?

Reply:
La pin MTPM permet de switcher entre les 2 modes de consommation du capteur. En passant de LOW à HIGH, on effectue un reset

Voir Page 6 & 7
//www.dropbox.com/home/Development/hardware/electronics/datasheets?preview=Aceltis-MF4-Capacitive_Sensor-MTCH101.pdf

Reply:
D'accord, je vois, c'est dans un soucis de power management: un mode courant et un mode d'économie d'énergie  
Ok pour le garder tel quel

A garder en tête que ça n'a rien à voir avec un PowerOn et un PowerOff

Autre chose au sujet du reset: il faut que ce soit une autre fonction. TouchSensorKit ne doit pas savoir qu'il faille faire un `PowerMode::low` puis un `PowerMode::normal` pour faire un reset, surtout dans le cas spécifique du MTCH101.

```suggestion
	void setPowerMode(PowerMode power_mode);
	void reset();
```